File: application/src/Caravelo/Modules/Survey/Domain/Model/Answer.php
<?php

namespace Caravelo\Modules\Survey\Domain\Model;


class Answer
{

    /** @var  int $id */
    private $id;

    /** @var string $content */
    private $content;

    /** @var  \DateTime $createdAt */
    private $createdAt;

    /**
     * Answer constructor.
     * @param int $id
     * @param string $content
     * @param \DateTime $createdAt
     */
    public function __construct($id, $content, \DateTime $createdAt)
    {
        $this->id = $id;
        $this->content = $content;
        $this->createdAt = $createdAt;
    }

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @return string
     */
    public function getContent()
    {
        return $this->content;
    }

    /**
     * @return \DateTime
     */
    public function getCreatedAt()
    {
        return $this->createdAt;
    }
}

File: application-java/src/main/java/survey/domain/model/Answer.java
package survey.domain.model;

public class Answer {
	private String id;
	private String content;
	private String date;
	
	public Answer(String id, String content, String date){
		this.id = id;
		this.content = content;
		this.date = date;
	}
	
	public String getId(){
		return this.id;
	}
	
	public String getContent(){
		return this.content;
	}
	
	public String getDate(){
		return this.date;
	}
}
